notebooks/203-meter-reader/203-meter-reader-with-paddle-ocr.ipynb

```diff
@@ -0,0 +1,1140 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "29b161d0-8144-4b3f-88fe-deb6abd21f1c",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "# Meter Reader with PaddleOCR\n",
+    "This notebook shows how to create a meter reader with OpenVINO Runtime. We use the pre-trained [PP-OCR](https://github.com/PaddlePaddle/PaddleOCR) to build up a inference task pipeline:\n",
+    "\n",
+    "1. Config the screen area of the meter reader.\n",
+    "2. Config the layout information of the meter reader.\n",
+    "3. Pre-process the image based on the given information.\n",
+    "4. Perform OCR recognition.\n",
+    "5. Structure output information.\n",
+    "\n",
+    "In this notebook, we recognize two type of meters. The biggest difference between two cases is that the display screens of the two devices are different. The case 1 is a field strength meter, the screen in the meter is not LCD, which means that we can read the pure text in the screen. The case 2 is a household electricity meter, the screen of the meter is LCD, which means we can note only see the text but also the outline of the text in the background.\n",
+    "\n",
+    "Their overall process of the two cases is similar, we only make some modifications in preprocessing for different meters. The following are the flowchart of two types of meter readers.\n",
+    "\n",
+    "<img align='center' src= \"https://user-images.githubusercontent.com/83450930/237536313-c6a26bb8-090c-4d09-8a35-31b96f7b5b04.png\" alt=\"drawing\" width=\"1500\"/>\n",
+    "\n",
+    "In some cases, the screen area in the image is not in a fixed position. A detection model can be used to dynamically provide the screen area information. Please see [PaddleDetection](https://github.com/PaddlePaddle/PaddleDetection) for more details.\n",
+    "\n",
+    "The tutorial consists of the following steps:\n",
+    "1. Prepare the PaddleOCR model.\n",
+    "2. Define configuration and helper functions\n",
+    "    - Prepare some configuration (take the case 1 as an example)\n",
+    "    - Define a helper function to apply affine transformation\n",
+    "    - Define helper functions for the preprocessing of text recognition\n",
+    "    - Define helper functions for the postprocessing of text recognition\n",
+    "3. Main Function with power frequency field strength meter\n",
```

Comment:
For purpose of general usage, please use the type of digital screen to identify these meter, other than the meter's scenario, so people can select the solution according to their customized scenario.